src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +102,112 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
+{{site.alert.note}}
 Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
```

Comment:
I find this to be a bit confusing. Is it correct to say that you run this from the directory where the pubspec is? Or to say that you run it from the top of the package?

Reply:
The top of the package should be the same location as the pubspec.
How about:
> When you change the Flutter plugin dependencies in `my_flutter/pubspec.yaml`,
run `flutter pub get` in your Flutter module directory to refresh the list
of plugins read by the `podhelper.rb` script. Then run `pod install` again from
in your application at`some/path/MyApp`.